index.html

```diff
@@ -1,21 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-<meta charset="utf-8" />
-<title>Dell Data Analytics Engine</title>
-</head>
-<body>
-<h1>Dell Data Analytics Engine</h1>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="refresh" content="0; URL=./latest/index.html" />
+    <title>Dell Data Analytics Engine</title>
+    <script type="text/javascript">
+      window.location.href = './latest/index.html';
+    </script>
+  </head>
+  <body>
+    <h1>Dell Data Analytics Engine</h1>
 
-<p>The following versions of the Dell Data Analytics Engine, powered by
-Starburst Enterprise platform (SEP) are available:
-</p>
+    <p>
+      The following versions of the Dell Data Analytics Engine, powered by
+      Starburst Enterprise platform (SEP) are available:
+    </p>
 
-<ul>
-  <li><a href="./435-e/index.html">435-e</a> (latest)</li>
-  <li><a href="./429-e/index.html">429-e</a></li>
-  <li><a href="./423-e/index.html">423-e</a></li>
-  <li><a href="./413-e/index.html">413-e</a></li>
-</ul>
-</body>
+    <ul>
+      <li><a href="./435-e/index.html">435-e</a> (latest)</li>
+      <li><a href="./429-e/index.html">429-e</a></li>
+      <li><a href="./423-e/index.html">423-e</a></li>
+      <li><a href="./413-e/index.html">413-e</a></li>
+    </ul>
```

Comment:
Should remove these contents of the page with a blanket "redirecting to latest documentation" message. No need to have a list of versions that would either go out of date or need to be updated for no reason.

Reply:
Probably with a static "Click here if you are not redirected" as a catchall.